bin/jsize

```diff
@@ -7,61 +7,123 @@ const pkg = require('../package.json')
 updateNotifier({ pkg: pkg }).notify()
 
 const chalk = require('chalk')
-const padEnd = require('pad-end')
+const spinner = require('ora')()
 const program = require('commander')
+const logUpdate = require('log-update')
 const prettyBytes = require('pretty-bytes')
-const spinner = require('ora')('Starting...').start()
+const stringWidth = require('string-width')
+const createTable = require('text-table')
 const jsize = require('../index')
+const originalLog = console.log
+const spacer = chalk.hidden('.') // Works better with table alignment than empty space.
```

Comment:
Actually looks like this in my terminal: 
![image](https://user-images.githubusercontent.com/141232/30001264-e7cbb65a-90b2-11e7-826d-ec70bbc237eb.png)

Reply:
Can copy-paste will be difficult